miq-dev-appliance-setup/MAC/on_appliance/source-updated.sh

```diff
@@ -38,7 +38,7 @@ cd $MIQ_DIR/vmdb || exit 1
 
 # Update gems.
 echo "**** Updating gems..."
-bundle update || exit 1
+bundle install --without qpid:metric_fu || exit 1
```

Comment:
I'm not sure if you need this in both places, see above.

I'm fine with it but it's due to change as we upgrade/change things.